CHANGELOG.md

```diff
@@ -3,12 +3,18 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## 1.0.0 - [date]
 
 Initial release of nf-core/readsimulator, created with the [nf-core](https://nf-co.re/) template.
 
 ### `Added`
 
+- Option to simulate amplicon sequencing reads
+- Option to simulate target capture sequencing reads
+- Option to simulate metagenomic sequencing reads
+- Option to simulate wholegenome sequencing reads
+- Samplesheet creation after reads have been simulated
```

Comment:
Can you add for which pipelines the samplesheet is created? Or is it just in general?

Reply:
The samplesheet is a general one similar to the samplesheet in the pipeline template where the headers are sample, fastq_1, and fastq_2. I know different nf-core pipelines have different requirements when it comes to samplesheets. Maybe the documentation should mention that it's just a general samplesheet.